2019/12xxx/CVE-2019-12254.json

```diff
@@ -1,18 +1,151 @@
 {
```

Comment:
CNA(s) of user (CERTVDE) do not match owning CNA.

Reply:
That's a pre-CNA CVE of cert@vde